### PR TITLE
feat: make samples self runnable by moving devrel tags

### PR DESCRIPTION
**Pull request by luongadam**
## Description

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [N/a] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [N/a] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [N/a] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [N/a] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: managedkafka/snippets/clusters/list_clusters.py [hunk 1]
```diff
@@ -12,23 +12,18 @@
...
-
-from google.cloud import managedkafka_v1
-
 
 def list_clusters(
     project_id: str,
     region: str,
-) -> List[str]:
-    """
-    List Kafka clusters in a given project ID and region.
```
Comment (acocuzzo) [inline managedkafka/snippets/clusters/list_clusters.py:26]: why remove this part?
Presumably it's still useful for testing the snippets?

File: managedkafka/snippets/clusters/get_cluster.py [hunk 2]
```diff
@@ -12,23 +12,20 @@
...
 
-# [START managedkafka_get_cluster]
-from google.cloud import managedkafka_v1
-
 
 def get_cluster(
     project_id: str,
     region: str,
     cluster_id: str,
-) -> managedkafka_v1.Cluster:
```
Comment (acocuzzo) [inline managedkafka/snippets/clusters/get_cluster.py:23]: same comment, I think we can leave the comments above the START statement

**Review by acocuzzo — APPROVED** [review]

File: managedkafka/snippets/clusters/list_clusters.py (hunk 1 above)
Comment (luongadam) [reply managedkafka/snippets/clusters/list_clusters.py:26]: That's true. I re-added the docstring outside of the devrel tags.

File: managedkafka/snippets/clusters/get_cluster.py (hunk 2 above)
Comment (luongadam) [reply managedkafka/snippets/clusters/get_cluster.py:23]: Done. I re-added the docstring outside of the devrel tags.

**Review by acocuzzo — APPROVED** [review]

**Pull request merged**